docs/how-to-setup-wsl.md

```diff
@@ -96,6 +108,18 @@ Once you have configured Docker Desktop to work with WSL2, follow these steps to
 
 We recommend you install the LTS release for Node.js with a node version manager - [nvm](https://github.com/nvm-sh/nvm#installing-and-updating).
 
+In your terminal, run the nvm installer like this:
+
+> [!NOTE]
+> We recommend updating to the latest stable releases of the software listed above, also known as Long Term Support (LTS) releases. However, some npm packages may not be compatible with newer versions of node. NVM can allow you to use npm packages with the compatible versions of node. Check your current node version before upgrading incase you need it later to work on a project outside of freeCodeCamp and notice compatibility issues
+
+```console
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.1/install.sh | bash
+
+# or
+
+wget -qO- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.1/install.sh | bash
+```
```

Comment:
This is just another pain-point of maintenance which is avoided by providing a link to the official NVM page - they keep their docs/version up to date.

```suggestion
```

Reply:
Thanks for the feedback. I was just trying to think of a away to speed up the process but it makes sense to refer to documentation that is up to date